cmd/imagestream.go

```diff
@@ -63,17 +67,30 @@ func (o *ImageStreamCleanupOptions) cleanupImageStreamTags(cmd *cobra.Command, a
 		return
 	}
 
-	commitHashes, err := git.GetCommitHashes(o.RepoPath, o.CommitLimit)
-	if err != nil {
-		log.WithError(err).WithField("RepoPath", o.RepoPath).WithField("CommitLimit", o.CommitLimit).Fatal("Retrieving commit hashes failed.")
+	var commits []string
+	if o.Tag {
+		if !git.IsValidSortValue(o.Sorted) {
+			log.WithField("namespace", o.Namespace).Fatal("Invalid sort flag")
+		}
+		var err error
+		commits, err = git.GetCommitTags(o.RepoPath, o.CommitLimit, git.SortTagBy(o.Sorted))
+		if err != nil {
+			log.WithError(err).WithField("RepoPath", o.RepoPath).WithField("CommitLimit", o.CommitLimit).Fatal("Retrieving commit tags failed.")
```

Comment:
logging with multiple fields can be combined `WithFields`, or move the chained method calls to next lines for readability.